### PR TITLE
Large card adjustments

### PR DESCRIPTION
**Pull request by breadadams**

### CODE REVIEW

File: src/components/Card/CardContent.js [hunk 1]
```diff
@@ -17,9 +20,7 @@ const CardContent = styled.div`
   min-width: 0;
   box-sizing: border-box;
 
-  ${props => props.large && css`
-    flex: 0 0 125px;
-  `}
+  ${props => props.cardSize && CardContentLarge}
 `
 
 const Title = styled.h2`
```
Comment (Kikobeats) [inline src/components/Card/CardContent.js]: not related but not sure why `h2`, can sense? can be `h1`? I don't know

File: src/components/Card/CardImage.js [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+import styled from 'styled-components'
+import { CardImageLarge } from './CardLarge'
 
 export default styled.div`
   display: block;
   flex: 0 0 125px;
-  background: ${props => (props.image ? `url(${props.image})` : ``)} no-repeat
-    center center / cover;
+  background: no-repeat center center / cover;
+  background-image: ${props => props.image ? `url(${props.image})` : ``};
```
Comment (Kikobeats) [inline src/components/Card/CardImage.js]: better this

```jsx
${props => props.image && css`
background-image: `url(${props.image})`
`}
```

File: src/components/Card/CardContent.js (hunk 1 above)
Comment (breadadams) [reply src/components/Card/CardContent.js]: Yeah I did a `h1` at first, but after inspecting an _existing_ embed (can't remember if Twitter or FB), they used `h2` (I guess for SEO maybe?)

File: src/components/Card/index.js [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+import { getUrlPath } from '../utils'
+
+export type CardSizes = 'large' | 'small'
 
 type CardProps = {
+  className?: string,
   contrast?: boolean,
-  endpoint?: string,
-  url: string,
   endpoint: string,
```
Comment (Kikobeats) [inline src/components/Card/index.js]: this is optional. The only required paramter is `url`

File: src/components/Card/CardImage.js (hunk 2 above)
Comment (breadadams) [reply src/components/Card/CardImage.js]: Done, with some deconstructing 👌

File: src/components/Card/index.js (hunk 3 above)
Comment (breadadams) [reply src/components/Card/index.js]: This is from flow @Kikobeats https://flow.org/en/docs/react/components/#toc-using-default-props-for-functional-components

> Note: You don’t need to make foo nullable in your Props type. Flow will make sure that foo is optional if you have a default prop for foo.

Comment (Kikobeats) [reply src/components/Card/index.js]: 👍

**Pull request merged**